pages/topics/experiments.mdx

```diff
@@ -433,9 +433,15 @@ A fingerprint will only be generated and returned if no authorization or fingerp
 
 ###### Query String Parameters
 
-| Field                   | Type    | Description                                               |
-| ----------------------- | ------- | --------------------------------------------------------- |
-| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data |
+| Field                   | Type    | Description                                                           |
+| ----------------------- | ------- | --------------------------------------------------------------------- |
+| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data             |
+| platform?               | string  | Supplements the response with experiments from the given [platform](#experiment-platform) |
```

Comment:
```suggestion
| platform?               | string  | Whether to also include experiments for the given [platform](#experiment-platform) |
```

It's still unclear to me what the param actually does... 

Maybe worth it to add a sentence outside the table to clarify in what situation it's sent and why the data isn't included by default?